examples/text-generation/run_generation.py

```diff
@@ -206,6 +206,13 @@ def setup_parser(parser):
     )
     parser.add_argument("--verbose_workers", action="store_true", help="Enable output from non-master workers")
 
+    parser.add_argument(
+        "--kv_cache_fp8",
+        action="store_true",
+        help="Store kv-cache in float8 when kv-cache is used",
+    )
+    parser.add_argument("--enable_quantization", "-eq", action="store_true", help="Enable Quantization to fp8")
```

Comment:
Having 2 arguments seems a bit redundant, can't we keep only `--kv_cache_fp8`? Or do they have different meaning?

Reply:
they have different meaning,
enable_quantization - enable syanpse quantization in the bridge, setting all gemm operations to fp8.
kv_cache_fp8 - is only to store kv cache in fp8 - should be used when enable_quantization is turn on but it's not a must. it's saves memory and has a better perf.

Reply:
Okay I see, in that case I would rename `enable_quantization` by `fp8`. Or will it also enable fp4 in the future?

Reply:
let's stick with enable_quantization

Reply:
IMO `enable_quantization` is misleading as there are tons of different quantization format and strategies available in the HF ecosystem. People may think that int8 or fp4 quantization is available for instance.
We also use "fp8" in Accelerate for instance: https://huggingface.co/docs/accelerate/v0.24.0/en/package_reference/kwargs#accelerate.utils.FP8RecipeKwargs
So, unless there is a strong argument for sticking to `enable_quantization`, I would prefer to have a more explicit name.

Reply:
fixed